db/migrate/20230913152019_add_questions_to_consent_form.rb

```diff
@@ -0,0 +1,10 @@
+class AddQuestionsToConsentForm < ActiveRecord::Migration[7.0]
+  def change
+    add_column :consent_forms,
+               :health_questions,
+               :jsonb,
+               null: false,
+               default: {
+               }
```

Comment:
In the controller you're using a number to access the answer here, e.g. `1`. Should this be a `[]`? Or are you future proofing?

Reply:
```ruby
      health_questions[current_health_question_index].merge! health_question_params
```

https://github.com/nhsuk/record-childrens-vaccinations/pull/499/files#diff-e0e1a4b64eb811d449a4d96f396f2136a5457ddcc8006c8551831caf75ac36fbR25

Reply:
You're right, it should be `[]`